benchmarks/benchmark_serving.py

```diff
@@ -2,8 +2,8 @@
 
 On the server side, run one of the following commands:
     vLLM OpenAI API server
-    python -m vllm.entrypoints.openai.api_server \
-        --model <your_model> --swap-space 16 \
+    vllm serve <your_model> \
```

Comment:
Q for backward compatibility: previous command still works?

Reply:
yup works exactly as is.